### DISCUSSION TITLE
Handheld console set up [retroarch] [question]

### DISCUSSION THREAD

**Discussion by bakeiro** [nzp-team/nzportable#673] [General]
Hi!

I have a R36S console (linux console using retroArch with a lot of emulators) I was wondering how I could run this natively since it's a Linux console, (I guess I could use an emulator like PSP but some emulators don't run 100% nice)

any help/tip with this?

thanks in advance!!

**Comment by MotoLegacy** [comment]
Hi there, for native play on the R36S, being an AARCH64 system, `nzportable-linuxarm64.zip` is going to be the binary release package you'll want. It is important to note, however, that "Desktop" releases of NZ:P do not have any sort of controller functionality at this time (see #362). You'd have to bind keyboard+mouse actions to the controller input on the system for this reason, if you want to avoid emulation.

**Reply by bakeiro** [reply]
I see, thanks! (I'm still struggling to figure out how I open the binary from RetroArch since I don't have a terminal and for opening a game I need a core (which is related with emulators)

Anyhow without controller input doesn't make sense... Thanks for the info!!
